peppy/sample.py

```diff
@@ -184,6 +185,17 @@ def __init__(self, series, prj=None):
         # analysis time, and a pipeline author vs. a pipeline user).
         self.paths = Paths()
 
+    # The __reduce__ function provides an interface for
+    # correct object serialization with the pickle pickle.
+    def __reduce__(self):
+        return (
+            self.__class__,
+            (self._series,),
```

Comment:
Is the idea with providing just the original (passed to the constructor) data, rather than any data that's been added since construction, that anything that's not passed in via the series is more dynamic/ephemeral and therefore shouldn't be preserved?